project/src/components/location-nav/location-nav.tsx

```diff
@@ -1,38 +1,33 @@
+import { useAppDispatch, useAppSelector } from '../../hooks';
+import { changeCity } from '../../store/actions';
+import { CITIES } from '../../const';
+
 export default function LocationNav() {
+
+  const currentCity = useAppSelector((state) => state.currentCity);
+  const dispatch = useAppDispatch();
+
+
   return (
     <div className="tabs">
       <section className="locations container">
         <ul className="locations__list tabs__list">
-          <li className="locations__item">
-            <a className="locations__item-link tabs__item" href="todo">
-              <span>Paris</span>
-            </a>
-          </li>
-          <li className="locations__item">
-            <a className="locations__item-link tabs__item" href="todo">
-              <span>Cologne</span>
-            </a>
-          </li>
-          <li className="locations__item">
-            <a className="locations__item-link tabs__item" href="todo">
-              <span>Brussels</span>
-            </a>
-          </li>
-          <li className="locations__item">
-            <a className="locations__item-link tabs__item tabs__item--active" href="todo">
-              <span>Amsterdam</span>
-            </a>
-          </li>
-          <li className="locations__item">
-            <a className="locations__item-link tabs__item" href="todo">
-              <span>Hamburg</span>
-            </a>
-          </li>
-          <li className="locations__item">
-            <a className="locations__item-link tabs__item" href="todo">
-              <span>Dusseldorf</span>
-            </a>
-          </li>
+          {CITIES.map((city) =>{
```

Comment:
Не забывай про пробелы
```suggestion
          {CITIES.map((city) => {
```